tools/pipelines/templates/1ES/build-npm-package.yml

```diff
@@ -194,7 +194,9 @@ extends:
               # over time and it might be necessary to bump it.
               # AB#6680 is also relevant here, which tracks rethinking how and where we run tests (likely with
               # a focus on e2e tests)
-              timeoutInMinutes: 60
+              # Note, This was recently updated to 90 minutes to account for the additional build time added from extending 
+              # the Microsoft 1ES template required for corporate security compliance
+              timeoutInMinutes: 90
```

Comment:
Note for reviewers: this is already in the original `build-npm-package.yml`, it's not a change required only in the 1ES version.